Patch-Source: http://patch-tracker.debian.org/patch/series/view/bzr-fastimport/0.13.0-4/disable-known-graph-feature.patch

Description: Disable know graph feature.
 repo.get_known_graph_ancestry() can't be called while in the middle of a write
 group. Otherwise bzr will crash with: 'BTreeBuilder' object has no attribute
 '_find_ancestors'.
Author: Felipe Contreras <felipe.contreras@gmail.com>
Bug: https://launchpad.net/bugs/541626
Bug-Ubuntu: https://launchpad.net/bugs/541626

--- a/revision_store.py
+++ b/revision_store.py
@@ -170,7 +170,7 @@
         """
         self.repo = repo
         self._graph = None
-        self._use_known_graph = True
+        self._use_known_graph = False
         self._supports_chks = getattr(repo._format, 'supports_chks', False)
 
     def expects_rich_root(self):
